### PR TITLE
Write performance section of v2.1 announcement

### PR DESCRIPTION
**Pull request by dominiklohmann**
This fills out the first section of the v2.1 blog post with our performance measurements. Note that the blog post is still in draft mode, we won't publish it until the actual release is out.

### :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Read it.

### CODE REVIEW

File: web/blog/vast-v2.1/index.md [hunk 1]
```diff
@@ -8,25 +8,47 @@ tags: [release, performance]
 ---
 
 [VAST v2.1][github-vast-release] is out! This release comes with a particular
-focus on performance, and brings a new utility for optimizing databases in
-production.
+focus on performance and reducing the size of VAST databases, and brings a new
+utility for optimizing databases in production.
```
Comment (dispanser) [inline web/blog/vast-v2.1/index.md:12]: This reads a bit rough to me, with the `and ... and,`. Maybe we can split that into two sentences, potentially circling back in that the tool allows us to take full advantage of these changes and improvements also for existing databases?

**Review by dispanser — COMMENTED** [review]
Reads good to me. A minor nitpick on what I believe is a missing digit and a bit of wording. I'm fully aware that I'm not the expert in these questions, so take my recommendation with a grain of salt :).

File: web/blog/vast-v2.1/index.md [hunk 2]
```diff
@@ -8,25 +8,47 @@ tags: [release, performance]
...
+To benchmark this, we used [`speeve`][speeve] to generate 20 Eve JSON files
+containing 8,388,608 events each, totalling 167,772,160 events. We spawned a
+VAST server process and ran 20 VAST client processes importing one file each in
+parallel. The numbers speak for themselves:
+
+|Version|v2.0.0|v2.0.0|v2.1.0-rc2|
+|-:|:-|:-|:-|
+|Configuration|v2.0 defaults|v2.1 defaults|v2.1 defaults|
+|Ingest Duration|1,650s|240s|242s (-85.3%)|
+|Ingest Rate|101,680/s|699,05/s|693,273/s (+581.8%)|
```
[the remark this answers is not in the document]
Comment (dominiklohmann) [reply web/blog/vast-v2.1/index.md]: This is already fixed up, I deleted one digit too many when cutting off the decimal places.

File: web/blog/vast-v2.1/index.md [hunk 3]
```diff
@@ -8,25 +8,60 @@ tags: [release, performance]
 ---
 
 [VAST v2.1][github-vast-release] is out! This release comes with a particular
-focus on performance, and brings a new utility for optimizing databases in
-production.
+focus on performance and reducing the size of VAST databases. It brings a new
+utility for optimizing databases in production, allowing existing deployments to
+take full advantage of the improvements after updating.
```
Comment (mavam) [inline web/blog/vast-v2.1/index.md:13]:
```suggestion
take full advantage of the improvements after upgrading.
```

File: web/blog/vast-v2.1/index.md [hunk 4]
```diff
@@ -8,25 +8,60 @@ tags: [release, performance]
...
+Additionally, VAST now compresses on-disk indexes with Zstd, resulting in a
+50-80% size reduction depending on the type of indexes used.
+
+This allowed us to increase the default partition size from 1,048,576 to
+4,194,304 events[^1], and the default number of events in a single batch from 1,024
+to 65,536, resulting in a massive performance increase at the cost of a ~20%
+larger memory footprint at peak load. Use the option `vast.max-partition-size`
+to tune this space-time tradeoff.
+
+To benchmark this, we used [`speeve`][speeve] to generate 20 Eve JSON files
```
Comment (mavam) [inline web/blog/vast-v2.1/index.md:35]:
```suggestion
To benchmark this, we used [`speeve`][speeve] to generate 20 EVE JSON files
```

**Review by mavam — APPROVED** [review]
Nice! Left a few nit-picky things.

**Pull request merged**